### PR TITLE
Package Updates

### PR DESCRIPTION
**Pull request by zSoulweaver**
Updates all the outdated packages in project, main package change is for Nuxt moving from `2.14.12` to `2.15.3`. Nuxt 2.15.x also updates the nuxt/components package to v2, which has some neat changes.

The new components package will now prefix components with their directory structure so there shouldn't be any need to do the `-components` folder trick we are using currently to scope components to pages. See the [migration guide](https://github.com/nuxt/components#migration-guide) for some detailed info. A second PR will be going up shortly that moves all the components to the global components directory.

The new nuxt update must also do something weird with the naming of components as the `LazyImage` component kept throwing a warning that wasn't very helpful... and took for ages to track down.
`[Vue warn]: Do not use built-in or reserved HTML elements as component id: Image`
I'd imagine it must be removing the `Lazy` portion of the component name and leaves it with just Image, but it's weird that it didn't happen before, I can't see anything in the change logs that would indicate a change in this behavior. Regardless, the component has now been renamed to `ImageLazy` and I've updated all the uses of the old component in the content directory.

So just a heads up for the people making blog content (@Moonded and co) that `LazyImage` must now be a `ImageLazy` tag going forward, soz fam.

### CODE REVIEW

**Comment by kpracuk** [comment]
"LazyImage" component problem is connected to dynamic imports.
If you prefix component name with "Lazy" it becomes dynamically imported component.
[https://nuxtjs.org/docs/2.x/directory-structure/components#dynamic-imports](https://nuxtjs.org/docs/2.x/directory-structure/components#dynamic-imports)
TLDR
You have `SomeComponent.vue`, you can now use `<some-component>` to load it 'normally' with a bundle or `<lazy-some-component>` to load it dynamically. Thats why it assumes there is a `Image.vue` somewhere in the code because it is called by `<lazy-image>`.
`<lazy-lazy-image>` should do the trick but it looks weird and is forced to load dynamically.

**Comment by kpracuk** [comment]
Tested it locally, haven't seen any problems. LGTM

**Review by kpracuk — APPROVED** [review]

**Comment by zSoulweaver** [comment]
> "LazyImage" component problem is connected to dynamic imports.
> If you prefix component name with "Lazy" it becomes dynamically imported component.

I figured that might of been the case. Do you have any other suggestions for the component name or is ImageLazy fine?
